### DISCUSSION TITLE
LIghthouse:Accessibility issue on webpage - no clues where issue is

### DISCUSSION THREAD

**Discussion by jasonc3107** [GoogleChrome/lighthouse#16233] [Q&A]
I am using lighthouse in chrome, under accessibility section it states...
Background and foreground colours have a sufficient contrast ratio Error!
Yet does not say 'where' the issue is.
Please can someone advise how I get more information on the location of the issues that lighthouse is saying.

**Comment by SollyBunny** [comment]
![image](https://github.com/user-attachments/assets/2d4fb1d8-af73-443c-8cad-8fe72e8c5ea1)
Still an issue
